merger/mergers.py

```diff
@@ -14,7 +15,8 @@ class BaseMerger:
     def __init__(self, clients):
         try:
             self.aspace_helper = ArchivesSpaceHelper(clients["aspace"])
-            self.cartographer_client = clients["cartographer"]
+            if settings.CARTOGRAPHER['cartographer']:
+                self.cartographer_client = ArchivesSpaceHelper(clients["cartographer"])
```

Comment:
```suggestion
            self.cartographer_client = clients.get("cartographer")
```
This won't throw a KeyError, and will then just set `self.cartographer_client` to `None` which means we can then test against that attribute rather than importing settings and testing against that.

Reply:
That works for me (I'd actually used CARTOGRAPHER_USE on the first pass).